sklearn/decomposition/pca.py

```diff
@@ -120,6 +120,10 @@ class PCA(_BasePCA):
     Notice that this class does not support sparse input. See
     :class:`TruncatedSVD` for an alternative with sparse data.
 
+    Although Bessel correction decreases the bias of the estimator, it also
```

Comment:
Without a reference this indeed seems odd...  +1 for your quoted wording. It feels clearer to me.